src/components/Accordion/components/Default.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Text } from '../../Text/Text';
+import { Box } from '../../Box/Box';
+import { Headline } from '../../Headline/Headline';
+import { Header } from './Header';
+import { ChevronUp } from './ChevronUp';
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
+    label: string;
```

Comment:
Same as before, are these types correct?

Reply:
Same,we can mark it as resolved